build.sbt

```diff
@@ -88,15 +88,8 @@ lazy val root = (project in file("."))
     Universal / javaOptions ++= Seq(
       "-Dconfig.file=/etc/gu/janus.conf", // for PROD, overridden by local sbt file
       "-Dpidfile.path=/dev/null",
-      "-J-XX:MaxRAMFraction=2",
```

Comment:
The error in the last attempt was thought to be caused by `"-J-XX:+PrintGCDateStamps"`, and there were warnings caused by the RAM fraction options`-J-XX:MaxRAMFraction=2` and `-J-XX:InitialRAMFraction=2`. The RAM fractions have been replaced with initial heap size(`Xms`) and max heap size (`Xmx`). After discussion with @adamnfish and @jacobwinch all the advanced options (`-XX`) were deemed unnecessary and thus removed.